satpy/etc/readers/fci_l2_nc.yaml

```diff
@@ -348,163 +417,147 @@ datasets:
     layer: 1
     standard_name: air_pressure_at_cloud_top
```

Comment:
Here I would add the `long_name` from the FCIL2FS but with the addition `"(lower layer)"` at the end

Reply:
appended "for upper/lower layer" to the long names